core/src/main/java/org/elasticsearch/search/suggest/DirectSpellcheckerSettings.java

```diff
@@ -130,4 +130,20 @@ public void minDocFreq(float minDocFreq) {
         this.minDocFreq = minDocFreq;
     }
 
+    @Override
+    public String toString() {
+        return "[" +
```

Comment:
Right now, `DirectSpellcheckerSettings` does not extend `ToXContentToBytes`.  Do you mean anywhere we want a `toString()` that outputs as JSON, that class should extend `ToXContentToBytes`?